### DISCUSSION TITLE
Kalendar Events Problem

### DISCUSSION THREAD

**Discussion by budiviper** [hi-manshu/Kalendar#103] [Q&A] [answered]
Hi guys, anybody know why when i tried to make Kalendar events like snippets below

`Box(modifier = Modifier.fillMaxSize().background(color = bgColor)
    ){
        Kalendar(
            kalendarType = KalendarType.Firey,
            kalendarEvents = listOf(
                KalendarEvent(LocalDate(2023, 3, 7), "BTS Concert")
            ),
            kalendarThemeColor = (KalendarThemeColor(kalBgColor,kalDayColor,kalHeaderColor)),
            kalendarDayColors = (KalendarDayColors(kalTextColor,kalSelectedColor))
        )
    }`

my expectation it should only appears 1 dot event on 7 march 2023,  but dot event appears redundant on all days, why is that?

**Comment by hi-manshu** [comment, marked as answer]
Hey Hey!
Its fixed
